packages/algob/src/lib/script-checkpoints.ts

```diff
@@ -18,6 +18,7 @@ import {
 } from "../types";
 
 export const scriptsDirectory = "scripts";
+export const testsDirectory = "test";
```

Comment:
Please rename this to `tests`

Reply:
I was confused with this, but then I found the tests folder named as `test` in crowdfunding example. 
I'll change them both appropriately.

Reply:
we can have multiple files in `test`. therefore `tests` is more "intuitive"

Reply:
but does it work ?

Reply:
> we can have multiple files in `test`. therefore `tests` is more "intuitive"

Okay, make sense. Let's add somewhere in `docs` (or maybe in algob help) that test directory should be named `tests` then. I don't think tests will run if user adds/renames it to test

Reply:
But usually in all packages i find the test folder to be named as "test" (instead of "tests", even with algob & runtime as well).

Reply:
@robert-zaremba  What do you think?

Reply:
I think both works, it's a small  detail. Whatever you decide will work, let's just be consistent.

Reply:
I think I agree with Amit here, `tests`  is the one we should use. 
I'll update the name of the directories, then.

Reply:
I am changing this to `test` as most test packages like mocha or jest, look for the test files in that.